### ISSUE TITLE
Duplicated response ID

### ISSUE THREAD

**Issue by claudioperez** [OpenSees/OpenSees#1493]
The number `10` appears to be used twice after `"energy"` was recently changed from `2000`. Changing this to use an enum can  help a lot with avoiding this kind of bug.

https://github.com/OpenSees/OpenSees/blob/032a6a79c8dd041e8fbee957296a75da656205e6/SRC/element/forceBeamColumn/ForceBeamColumn3d.cpp#L3660C14-L3660C31

**Comment by mhscott** [comment]
Please make a PR to fix this

**Comment by MassimoPetracca** [comment]
@claudioperez Yes I recently changed the index 10 for "energy" to 2000 because it was already used for another response.
However, I just noticed that it has been changed back to 10 in PR #1439. Probably unintentionally, his branch was not up-to-date with the master.

@mhscott should we open a PR to change the index from 10 to a unique one? or is there a reason for it to be changed back to 10?

**Comment by mhscott** [comment]
@MassimoPetracca Yeah, go ahead and fix it. Thanks!

**Comment by MassimoPetracca** [comment]
Done in PR #1502
